code/game/objects/structures/flora.dm

```diff
@@ -94,13 +99,36 @@
 	death()
 	return ..()
 
-/obj/structure/flora/pottedplant/attackby(obj/item/weapon/W, mob/user)
-	if (W.edge)
-		user.visible_message(span("warning", "[user] cuts down the [src]"))
-		death()
-		return 1
+/obj/structure/flora/pottedplant/attackby(obj/item/W, mob/user)
+	user.visible_message("[user] begins digging around inside of \the [src].", "You begin digging around in \the [src], trying to hide \the [W].")

```

Comment:
2nd sentence here seems awkwardly worded. Can't think of a better way to phrase it off the top of my head though.

Reply:
Looks fine to me, tbh.